Launcher/Launcher/Launcher.csproj

```diff
@@ -0,0 +1,23 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
+        <ImplicitUsings>enable</ImplicitUsings>
+        <Nullable>enable</Nullable>
+	    <RuntimeIdentifiers>win-x64;win-arm64</RuntimeIdentifiers>
+	    <AssemblyName>NixOS</AssemblyName>
+	    <Version>1.3.3.7</Version>
```

Comment:
Use a real version. Look at something like [`GitVersion`][1], [`Nerbank.Gitversioning`][2], or [`Minver`][3] if you don't want to manually manage it.

[1]: https://github.com/GitTools/GitVersion
[2]: https://github.com/dotnet/Nerdbank.GitVersioning
[3]: https://github.com/adamralph/minver

Reply:
The `1.3.3.7` is just a placeholder, it gets replaced by the CI script. From what I understand, this may only be a numeric value, because otherwise I'd have used `DEV_BUILD` here